### ISSUE TITLE
About loss used

### ISSUE THREAD

**Issue by bkumardevan07** [as-ideas/TransformerTTS#60]
is your tts loss average of (absolute error, cross entropy, mean squared error...) ???
Can you tell me what is actual tts loss used in literature. Actual I was looking but unable to find.
Thanks

**Comment by cfrancesco** [comment]
Which model?
L1 for forward, L1 (mels) and crossentropy (stop) for autoregressive.
https://github.com/as-ideas/TransformerTTS/blob/master/model/models.py inside of _compile methods.

**Comment by bkumardevan07** [comment]
but in losses.py you are adding all the losses and then passing the sum , right??
for autoregressive model
